count_server/web/router.ex

```diff
@@ -16,11 +16,17 @@ defmodule CountServer.Router do
   scope "/", CountServer do
     pipe_through :browser # Use the default browser stack
 
-    get "/", CounterController, :index
+    # get "/", CounterController, :index
+    get "/", HomeController, :index
   end
 
   # Other scopes may use custom stacks.
-  # scope "/api", CountServer do
-  #   pipe_through :api
-  # end
+  scope "/api/v1", CountServer do
```

Comment:
This is something i did too, and I don't know if its the best choice.
The url is scoped by /api/v1, but not the path helpers or the module names.
Will this be a problem in a future v2 of the api? I have no idea. 
Will this create conflicts if you need a different SessionController for the browser and the api pipelines? Probably

As I said I don't know if it is going to affect, but in rails it probably wil